apps/mudita-center-e2e/src/specs/settings/mc-version-check-for-updates.e2e.ts

```diff
@@ -7,54 +7,57 @@ import SettingsPage from "../../page-objects/settings.page"
 import NavigationTabs from "../../page-objects/tabs.page"
 import ModalPage from "../../page-objects/mc-update-modal.page"
 import HomePage from "../../page-objects/home.page"
+import TestHelper from "../../helpers/tests.helper"
+
+if (TestHelper.isNotLinux()) {
+  describe("Checking for Mudita Center updates", () => {
+    before(async () => {
+      const notNowButton = await HomePage.notNowButton
+      await notNowButton.waitForDisplayed()
+      await notNowButton.click()
+    })
```

Comment:
```suggestion
describe("Checking for Mudita Center updates", () => {
  before( async function(){
    this.skip()
    const notNowButton = await HomePage.notNowButton
    await notNowButton.waitForDisplayed()
    await notNowButton.click()
  })
```

Using the `.skip()` method offers two key benefits:
1. **Report Visibility:** Skipped tests are clearly marked in the report, facilitating test management.
2. **Code Readability:** Avoiding nesting improves the readability and structure of the test code.

For more details: [Mocha - Inclusive Tests](https://mochajs.org/#inclusive-tests).

Reply:
Changed